pkg/types/cciptypes/models.go

```diff
@@ -0,0 +1,38 @@
+package cciptypes
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/common/hexutil"
+)
+
+type Address string
+
+func (a Address) Equals(addr2 Address) bool {
+	if common.IsHexAddress(string(a)) && common.IsHexAddress(string(addr2)) {
```

Comment:
ah, i see, given that Address is now a string, we can change this to return a == addr2

Reply:
Code in this repo cannot make any assumptions about the format though. It has to work with any chain implementation, and it is their responsibility to format the strings canonically.

Reply:
you could add this functionality as a helper func in the evm relayer if you need it.